Add passwordAuthenticationmethod and resetPassword to V1 (#17299)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/passwordauthenticationmethod-get.md

---
title: "Get passwordAuthenticationMethod"

> description: "Retrieve the properties and relationships of passwordauthenticationmethod object."

Reviewer: ```suggestion
description: "Retrieve a user's passwordAuthenticationMethod object."
```

ms.localizationpriority: medium
author: "mmcla"
ms.prod: "identity-and-sign-in"
doc_type: "apiPageType"
---

# Get passwordAuthenticationMethod

Namespace: microsoft.graph

> Retrieve the properties and relationships of a [password authentication method](../resources/passwordauthenticationmethod.md) object. 

Reviewer: Same logic as the List doc suggestion
```suggestion
Retrieve a password that's registered to a user, represented by a [passwordAuthenticationMethod](../resources/passwordauthenticationmethod.md) object. For security, the password itself will never be returned in the object and the **password** property is always `null`.
```

### Permissions acting on self

|Permission type      | Permissions (from least to most privileged)              |
|:---------------------------------------|:-------------------------|
| Delegated (work or school account)     | UserAuthenticationMethod.Read, UserAuthenticationMethod.ReadWrite |
| Delegated (personal Microsoft account) | Not supported. |
| Application                            | Not supported. |

### Permissions acting on other users

|Permission type      | Permissions (from least to most privileged)              |
|:---------------------------------------|:-------------------------|
| Delegated (work or school account)     | UserAuthenticationMethod.Read.All, UserAuthenticationMethod.ReadWrite.All |
| Delegated (personal Microsoft account) | Not supported. |
| Application                            | UserAuthenticationMethod.Read.All, UserAuthenticationMethod.ReadWrite.All |

For delegated scenarios where an admin is acting on another user, the admin needs one of the following [Azure AD roles](/azure/active-directory/users-groups-roles/directory-assign-admin-roles#available-roles):

> * Global administrator
> * Global reader
> * Privileged authentication administrator
> * Authentication administrator (only sees masked phone numbers)

Reviewer: Same logic as the List doc
```suggestion
* Global Administrator
* Global Reader
* Privileged Authentication Administrator
* Authentication Administrator
```
